monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java

```diff
@@ -322,7 +328,28 @@ public ScheduleDefinition eval(InputStream inputStream) {
 
         return schedule;
     }
+    
+    private ScheduleDefinition getScheduleWithoutProperties(final TenantIdentifier tenant, 
+            final JobIdentifier jobIdentifier) {
+        final JobContext jobContext = getJobContext(tenant, jobIdentifier);
+        final String groupName = jobContext.getGroupName();
+        final ScheduleDefinition schedule = new ScheduleDefinition(tenant, jobIdentifier, groupName);
```

Comment:
The old functionality is still there with the second argument set to "true". But I have to check hot folder.